bigquery/rest/src/test/java/com/example/bigquery/Constants.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not  use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http=//www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.example.bigquery;
+
+public class Constants {
+  public static final String PROJECT_ID = "cloud-samples-tests";
```

Comment:
Removed. Yeah, it's lazy to hard-code it. It wasn't much harder to just look at the semi-standard GOOGLE_CLOUD_PROJECT environment variable, so I updated the tests to look at that.